src/test/java/uk/gov/digital/ho/pttg/api/AuditHistoryResourceRecordsIntTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.digital.ho.pttg.api;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonpath.JsonPath;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlGroup;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+import static org.springframework.test.context.jdbc.Sql.ExecutionPhase.AFTER_TEST_METHOD;
+import static org.springframework.test.context.jdbc.Sql.ExecutionPhase.BEFORE_TEST_METHOD;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = RANDOM_PORT)
+@SqlGroup({
+        @Sql(executionPhase = BEFORE_TEST_METHOD, scripts = "/sql/AuditHistoryRecordsIntTest/before.sql"),
+        @Sql(executionPhase = AFTER_TEST_METHOD, scripts = "/sql/AuditHistoryRecordsIntTest/after.sql")
+})
+public class AuditHistoryResourceRecordsIntTest {
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Test
+    public void shouldRetrieveRecords() throws IOException {
+        String url = "/records?correlationId=correlation-id-1&eventTypes=INCOME_PROVING_FINANCIAL_STATUS_REQUEST,INCOME_PROVING_FINANCIAL_STATUS_RESPONSE";
```

Comment:
This url will also become "/history" if you change the resource as suggested by Mike

Reply:
I've had to make it /historyByCorrelationId (Spring doesn't let you have two methods annotated as the the same URL and putting both into one method will be ugly) but I have updated the tests to match